dashboard/db/migrate/20170118223108_change_best_result_for_contained_levels.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeBestResultForContainedLevels < ActiveRecord::Migration[5.0]
+  def change
+    # Find all existing UserLevels for contained levels, and update their best
+    # result to be CONTAINED_LEVEL_RESULT (101)
+    containers = Level.all.select{|x| !x.contained_levels.empty?}
+    contained_level_ids = containers.map{|x| x.contained_levels[0].id}
+    UserLevel.where(level_id: contained_level_ids).update_all(best_result: 101)
```

Comment:
This is a (almost) non-indexed scan, requiring a full scan of UserLevel. Are we fine with taking down production for some length of time? If not, this is going to need to be implemented differently.